pkg/testschema/table.go

```diff
@@ -91,3 +91,41 @@ func (t *Table) Init(s *Schema, c QueryCache) {
 	t.queryCache = c
 	t.queryCache.BindToTable(t)
 }
+
+func (t *Table) HaveValidColumnsForDelete() bool {
+	return len(t.ValidColumnsForDelete()) != 0
+}
+
+func (t *Table) ValidColumnsForDelete() []int {
```

Comment:
Change it to return []*ColumnDef

Reply:
In case with '[]*ColumnDef', in 'PostStmtHook'  function for delete selected column will need searching idx under 'Columns' array of 'Table' struct.
`		PostStmtHook: func() {
			t.Columns = t.Columns.Remove(colNum)
			t.ResetQueryCache()
		},`

Reply:
Sure, just update Remove method